nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs

```diff
@@ -33,8 +33,6 @@ public class ItemView<TViewModel> : VanillaItemView
 #endif
         public TouchHandler touchHandler;
 
-        public Button itemButton;
-        public Image backgroundImage;
```

Comment:
Is this intended?
If you intended it, these fields is not needs? Or controlled by other class?

Reply:
These are not used.

Reply:
👌